### DISCUSSION TITLE
Text alignment of TOC and Listing

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#6273] [Q&A] [answered]
### Description

I have modified the the styles.css to justify the text of the body.

The problem with this is that it also justifies the table of contents (toc) and also the listing of my posts. I would like to maintain the default alignment of the text in the toc (aligned to the left, not justified) and the same for the listing.

I have searched [here](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss) and in some forums but I have not found the way of doing that.

**Comment by cderv** [comment, marked as answer]
You should just use a CSS selector that target exactly what you need to justify. 
using this is too large
````css
body {
  text-align: justify;
}
````

Use a browser inspector to open the source of your page, and find the HTML element you really want to style. 

You could try something like
```css
main {
  text-align: justify;
}
```
 which would already not target the sidebar 

You could also be more precise like 
```css
main p {
  text-align: justify;
}
```

I would suggest 

* Learning a bit more about CSS and its selector
* Learning to use the browser inspector to look for current CSS rules, play with adding new one, and help know what to target 

A good url to learn a bit more about all this: https://rstudio4edu.github.io/rstudio4edu-book/css-crash.html
